src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -405,6 +405,109 @@ def on_request(self, request, **kwargs):
     return sdk_no_wait(no_wait, deployments_operation_group.create_or_update, resource_group_name, deployment_name, properties)
 
 
+def _deploy_arm_template_subscription_scope_unmodified(cli_ctx,
+                                                       template_file=None, template_uri=None,
+                                                       deployment_name=None, deployment_location=None,
+                                                       parameters=None, mode=None, validate_only=False,
+                                                       no_wait=False):
+    DeploymentProperties, TemplateLink = get_sdk(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES,
+                                                 'DeploymentProperties', 'TemplateLink', mod='models')
+    template = None
+    template_link = None
+    template_obj = None
+    if template_uri:
+        template_link = TemplateLink(uri=template_uri)
+        template_content = _urlretrieve(template_uri).decode('utf-8')
+        template_obj = _remove_comments_from_json(template_content)
+    else:
+        template_content = read_file_content(template_file)
+        template_obj = _remove_comments_from_json(template_content)
+
+    template_param_defs = template_obj.get('parameters', {})
+    template_obj['resources'] = template_obj.get('resources', [])
+    parameters = _process_parameters(template_param_defs, parameters) or {}
+    parameters = _get_missing_parameters(parameters, template_obj, _prompt_for_parameters)
+
+    template = json.loads(json.dumps(template))
+    parameters = json.loads(json.dumps(parameters))
+
+    properties = DeploymentProperties(template=template, template_link=template_link,
+                                      parameters=parameters, mode=mode)
+
+    smc = get_mgmt_service_client(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES)
+
+    class JsonCTemplate(object):
+        def __init__(self, template_as_bytes):
+            self.template_as_bytes = template_as_bytes
+
+    from msrest.serialization import Serializer
+
+    class MySerializer(Serializer):
```

Comment:
MySerializer. Naming could be improved.

Reply:
> MySerializer. Naming could be improved.

This is a temporary hack from Laurent, which will be removed and replaced with an official released library. Anyway, I can rename it with a more reasonable name.